src/utils/proxyMap.ts

```diff
@@ -1,4 +1,4 @@
-import { proxy } from 'valtio'
+import { proxy } from '../vanilla'
```

Comment:
I faced this issue with jotai/devtools yesterday, you may consider updating it also!

Reply:
*in vite

Reply:
what's the issue in jotai? jotai is tightly coupled with react for now. (I want to improve it eventually.)